### PR TITLE
stuff

### PR DESCRIPTION
**Pull request by elgreg**
???

### CODE REVIEW

**Review by asabaylus — COMMENTED** [review]
but @asabaylus should be listed first

File: README.md [hunk 1]
```diff
@@ -25,3 +25,6 @@ If you'd like to submit a talk idea, please create a [new GitHub issue](https://
 If you just want to ask a question, feel free to ing us on [Twitter](https://twitter.com/dcjavascript).
 
 You can also contact the organizers on the [Meetup](http://www.meetup.com/DC-JavaScript/) page.
+
+DCJS is run by @elgreg, @jakerella, and @asabaylus
+
```
Comment (asabaylus) [inline README.md:30]: new line???

Comment (asabaylus) [reply README.md:30]: Seriously force push ????

**Comment by jakerella** [comment]
:shipit: ;D

**Pull request closed without merging**